### ISSUE TITLE
Auto Refresh SMS Conversations

### ISSUE THREAD

**Issue by bkonia** [jpwalters/OpenVBX-Plugin-SMS-Conversation#32]
I'd like to suggest that the SMS conversations refresh automatically in real time, as new messages come in. The audio notifications work perfectly, so it should be pretty simple to do a browser refresh of the conversations.

**Comment by jpwalters** [comment]
Yes, this would be easy to do.  If I do this though what should happen to the highlighted row?  Right now I wait for the user to click the row before removing the highlight color.

**Comment by bkonia** [comment]
Yes, the messages should remain highlighted until the user clicks on them.
